.circleci/config.yml

```diff
@@ -53,3 +85,16 @@ workflows:
                 - main
     jobs:
       - release-management/test-ts-update
+  just-nuts:
+    when: << pipeline.parameters.run-just-nuts >>
+    jobs:
+      - release-management/test-nut:
+          name: just-nuts-<< matrix.os >>
+          sfdx_version: << pipeline.parameters.sfdx_version >>
+          sfdx_executable_path: sfdx
```

Comment:
What happens if you say `sfdx` ?  testkit has this https://github.com/salesforcecli/cli-plugins-testkit/blob/837c7e4c6d4956fac348a6ac6040bb1d45ee1e74/src/execCmd.ts#L94 and a different example on the readme, but this in the samples.  

Works ok?

Reply:
@mshanemc, I have set that locally and in that form it defers to PATH.

Reply:
@RodEsp the tests I have run in circleci seem to shows that it does.